dogfooding/ios/fastlane/Fastfile

```
@@ -78,7 +78,7 @@ platform :ios do
         groups: is_prod ? ['Internal Testers', 'External Testers'] : ['Internal Testers'],
         changelog: 'Lots of amazing new features to test out!',
         reject_build_waiting_for_review: false,
-        skip_waiting_for_build_processing: false,
+        skip_waiting_for_build_processing: true,
         beta_app_description: 'Beta build of Stream Video Flutter Dogfoofing app for internal and external testers',
```

RuboCop (1.76.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)